website/TRANSLATIONS.md

```diff
@@ -0,0 +1,39 @@
+# Contributing Guidelines for our docs
+
+Are you a native speaker of a language other than English? We'd love to have your help!
+
+## How to contribute
+
+(Read the [contribution guidelines](../CONTRIBUTING.md) first for general information about contributing to this project.)
+
+### Initial Translation
+
+If the docs are not available in your language yet and you would like to translate them, you can do so by following these steps:
+
+1. Copy the `src/content/docs` directory to a new directory (exclude all other translated code, e.g. `ja`, `pt-br`, etc.) with the name of your language, e.g. `src/content/docs/de` for German. (If you're not sure what the language code is, you can find it [here](https://en.wikipedia.org/wiki/List_of_ISO_639-1_codes).)
+
+2. File a draft PR to let others know you're working on it. This is to avoid duplicate work, and also allows for community feedback along the way.
+
+3. File by file, translate the content of the files.
```

Comment:
I'm not convinced this is the right approach for us. I think it's better to first create a new **issue** and track/coordinate to translate the pages, that's how we signal the intent of creating new translations. 

Also, it's best to create PRs for single pages, so we give the chance to other contributors to translate other pages.

Reply:
I think I need to adapt this part as well. So, it fits with our contribution culture.